Makefile

```diff
@@ -21,8 +21,11 @@ RELEASE_SRC = apache-apisix-ingress-controller-${VERSION}-src
 LOCAL_REGISTRY="localhost:5000"
 IMAGE_TAG ?= dev
 
+GITSHA ?= "No-git-module"
```

Comment:
All words in the output of `apisix-ingress-controller version` are in small case.

Reply:
done